pages/api/replicache-push.ts

```diff
@@ -99,16 +100,17 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
         }
 
         const next = push.mutations[i + 1];
-        console.log("Considering for collapse", mutation, next);
+        console.log("Considering for collapse", JSON.stringify(mutation, null, ''), JSON.stringify(next, null, ''));
```

Comment:
Just curious how you end up with both `"` and `'` on the same line? Have you not setup VSCode to format on save?

Reply:
No idea. What extensions do you use? I have prettier with the default settings.

Reply:
This line has been removed and there's no more single quotes in the file.

Reply:
`command` `,` -> `format on save` -> ✅

Reply:
I also have `esbenp.prettier-vscode`  installed if that matters.